app/Console/Commands/SyncBDclub.php

```diff
@@ -121,10 +112,13 @@ public function handle()
                 } else {
                     $extraData = json_decode("{\"isactive\":null,\"responsible\":\"\",\"music\":\"\",\"flyer\":\"\",\"flyerdone\":null,\"face\":\"\",\"facedone\":null,\"price_ak_n\":\"\",\"price_ak_v\":\"\",\"price_vk_n\":\"\",\"price_vk_v\":\"\",\"notes\":\"\"}");
                 }
+                if(is_null($extraData)){
+                    $extraData = json_decode("{\"isactive\":null,\"responsible\":\"\",\"music\":\"\",\"flyer\":\"\",\"flyerdone\":null,\"face\":\"\",\"facedone\":null,\"price_ak_n\":\"\",\"price_ak_v\":\"\",\"price_vk_n\":\"\",\"price_vk_v\":\"\",\"notes\":\"\"}");
```

Comment:
Why the json_decode here? Couldn't you just say something like:

```
$extraData = [ "isactive" => null, "responsible" => ""]  
```
and so on?

Reply:
i got a sample string from input, this data should every event provide in the comment section
if not i made a default string

Reply:
But why the json_decode? Couldn't you just set `$extraData` as an array? No need to decode a string.

Reply:
because i got the string already from the sample and i want to stay close to this
it was just copy and paste -> easyer and faster than rewrite it to a string